File: sportsshop-parent/sportsshop-cart-web/src/main/webapp/js/service/cartService.js
app.service('cartService', function($http){
	
	this.findCartList = function(){
		return $http.get('cart/findCartList.do?');
	}
	
	this.addGoodsToCartList = function(itemId, num){
		return $http.get('cart/addToCartList.do?itemId='+itemId+"&num="+num);
	}
	
});

File: sportsshop-parent/sportsshop-cart-interface/src/main/java/com/sportsshop/cart/service/CartService.java
package com.sportsshop.cart.service;

import java.util.List;

import com.sportsshop.pojo.TbUser;
import com.sportsshop.pojogroup.Cart;

public interface CartService {
	
	public List<Cart> addGoodsToCartList(List<Cart> cartList,Long itemId,Integer num);
	
	public List<Cart> findCartListFromRedis(Long userId);
	
	public void saveCartListToRedis(Long userId, List<Cart> cartList);
}
